.github/workflows/maven-publish.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will build a package using Maven and then publish it to GitHub packages when a release is created
+# For more information see: https://github.com/actions/setup-java/blob/main/docs/advanced-usage.md#apache-maven-with-a-settings-path
+
+name: Maven Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
**Suggestion:** The 'runs-on' field is set to 'ubuntu-latest'. It's a better practice to specify a particular version of Ubuntu to ensure the workflow is stable even if a new version of Ubuntu is released.
```suggestion
    runs-on: ubuntu-20.04
```